react/src/components/nplusone.js

```diff
@@ -0,0 +1,53 @@
+import { Component } from "react";
+import Context from "../utils/context";
+import { Link, useFetcher } from "react-router-dom";
+import * as Sentry from "@sentry/react";
+import { connect } from "react-redux";
+import { determineBackendUrl } from "../utils/backendrouter";
+class NPlusOneItem extends Component {
+  async componentDidMount() {
+    console.log("Mounting NPlusOneItem");
+    const response = await fetch(this.props.backend + "/product/0/info?id=" + this.props.index);
```

Comment:
Hey, in the event this errors (say a 500 response) - what happens to the NPlusOne component? Does it still render? Let's get some basic try/catch Sentry.captureException() style error handling in here.

Any chance you could test sending back a 500 response from the Flask end so we can see how it responds?

It's a lot of work now, but it will save us tons of headache debugging time if Flask is ever down or unexpected outcomes happen in the future.

Reply:
Yes. Currently getting 500s from the backend and the front end renders properly. I think this is ok.

Reply:
@bfpepper87 can you try to add some better erroring handling for all these failed responses that are appearing? If we could group them together that'd be sweet. No stacktraces on these so hard to make sense of what's happening. Not sure of it's the outbound http request or the http Response yet.
https://testorg-az.sentry.io/discover/homepage/?field=issue&field=title&field=count%28%29&name=All+Events&project=5808623&query=event.type%3Aerror+http.url%3Ahttps%3A%2F%2Fapplication-monitoring-react-dot-sales-engineering-sf.appspot.com%2Fnplusone&sort=-count&statsPeriod=7d&yAxis=count%28%29